setup.py

```diff
@@ -183,6 +183,7 @@ def get_git_sha() -> str:
         ],
         "oracle": ["cx-Oracle>8.0.0, <8.1"],
         "pinot": ["pinotdb>=0.3.3, <0.4"],
+        "playwright": ["playwright==1.37.0"],
```

Comment:
I think we try to avoid hard pinning in `setup.py` to minimize the risk of conflicts:
```suggestion
        "playwright": ["playwright=>1.37.0, <2"],
```